stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -93,6 +93,9 @@ spec:
           echo "Copying bootstrap configs";
           cp -Lrf /bootstrap/* /artifactory_extra_conf/;
           {{- end }}
+          mkdir -p {{ .Values.artifactory.persistence.mountPath }}/access/etc;
+          cp -Lrf /tmp/bootstrap.creds {{ .Values.artifactory.persistence.mountPath }}/access/etc/bootstrap.creds;
```

Comment:
Take it from `/tmp/access/bootstrap.creds`?

Reply:
Done